Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala

```diff
@@ -78,3 +82,25 @@ class ShivaBiopetplanet30xHg19Test extends ShivaSuccess with BwaMem with Hsapien
   override def vepVersion = Some("86")
   override def memoryArg = "-Xmx1G"
 }
+
+class ShivaNA12878WGSGRCh38Test extends ShivaSuccess with BwaMem with HsapiensGRCh38_no_alt_analysis_set with NA12878WGS
+  with Haplotypecaller with HaplotypecallerGvcf with Unifiedgenotyper {
+
+  override def disablescatter = false
+  override def memoryArg = "-Xmx4G"
+
+  def paired = true
+  def shouldHaveKmerContent = Some(true)
+
+  override def annotation = Some(true)
+  override def vepVersion = Some("86")
+
+  override def referenceVcf = Some(Biopet.fixtureFile("samples", "NA12878_wgs", "snp_indel_calls", "GIAB_GRCh38_v3.3.2.vcf.gz"))
+  override def referenceVcfRegions = Some(Biopet.fixtureFile("samples", "NA12878_wgs", "snp_indel_calls", "GIAB_GRCh38_regions_v3.3.2.bed"))
+
+  override def svCalling = Some(true)
+  val svCallers: List[String] = ShivaSvCalling.getSvCallersAsStrList(List(new Breakdancer, new Delly))
```

Comment:
Clever is working in the current develop branch. Could you add this in again?

ps: was waiting on this change to look into this PR ;)